pkg/ccr/handle/replace_table.go

```diff
@@ -0,0 +1,93 @@
+package handle
+
+import (
+	"fmt"
+
+	"github.com/cloudwego/kitex/tool/internal_pkg/log"
+	"github.com/selectdb/ccr_syncer/pkg/ccr"
+	"github.com/selectdb/ccr_syncer/pkg/ccr/record"
+	festruct "github.com/selectdb/ccr_syncer/pkg/rpc/kitex_gen/frontendservice"
+)
+
+func init() {
+	ccr.RegisterJobHandle[*record.ReplaceTableRecord](festruct.TBinlogType_REPLACE_TABLE, &ReplaceTableHandle{})
+}
+
+type ReplaceTableHandle struct {
+}
+
+func (h *ReplaceTableHandle) IsBinlogCommitted(j *ccr.Job, r *record.ReplaceTableRecord) (bool, error) {
+	return true, nil
```

Comment:
Should trigger fullsync, if the replace table binlog state is unknown

![image](https://github.com/user-attachments/assets/7dcfac25-9f3f-4b51-9c1c-95066e9dfe11)